lib/checks/shared/heading-whitespace-only.js

```diff
@@ -0,0 +1,19 @@
+if (axe.commons.text.accessibleTextVirtual(virtualNode)) {
+	return true;
+}
+
+const spaceSeparatorCharCodes = [
+	160, // non-breaking space
+	8194, // en space
+	8195, // em space
+	8201 // thin space
+];
+
+let hasBreakingSpaceElement = node.querySelectorAll('br, wbr').length > 0;
+let hasSpaceSeparator =
+	node.textContent
```

Comment:
@WilcoFiers - is there a better util method we can use instead of textContent.

@straker - does the `accessibleTextVirtual` trim out spaces? Perhaps worth passing an argument into it to not trim spaces and use that value from above here?

I remember using `textContent` in the past & it was asked to be changed (can't remember why).

Reply:
Ya,` accessibleTextVirtual` will return a 0 length string for all space separator characters